src/services/mentors.js

```diff
@@ -305,6 +309,7 @@ module.exports = class MentorsHelper {
 	 */
 	static async createMentorExtension(data, userId, orgId) {
 		try {
+			data.email = emailEncryption.encrypt(data.email.toLowerCase())
```

Comment:
@nevil-mathew Do consider allowing name, email and phone to be updated as well via the API